### DISCUSSION TITLE
Firefox won't download videos

### DISCUSSION THREAD

**Discussion by spikerdc** [aclap-dev/video-downloadhelper#1438] [Help and Q&A]
### Your question

I've always been able to download videos from a specific website, members only, in Firefox. Recently VideoDownloader shows MP4 0Kb in the left column where the blue button is to download. I can still download the videos on this same site on my laptop computer but not my Desktop anymore. Is there a solution so that i can download videos on my Desktop again. Interestingly, older videos that are posted are still downloadable on my Desktop. Thanks for any help.

### Addon & CoApp version and details

version: 9.1.0.44
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"04e08f723c1f4fbe"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:132.0) Gecko/20100101 Firefox/132.0
{
    "valid_license_message_has_been_displayed": true,
    "used_history_button": true,
    "download_directory": "C:\\Users\\Extreme PC TITAN\\Desktop",
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "C:\\Users\\Extreme PC TITAN\\Desktop"
        }
    },
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": true,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": true,
    "open_count_store": 6,
    "successfull_dl": 79,
    "never_show_successfull_dl_message": true,
    "history_limit_in_days": 5
}

**Comment by jcfrog** [comment]
Hi,
This should work the same on both computers. Maybe they changed streaming protocol and that's why old videos are still downloadable.
jerome

**Reply by spikerdc** [reply]
I dont' think that can be the case because on my laptop i can still download old videos or the new ones. Something had to have changed on my Desktop in firefox or with the Downloader ... at least that is my theory.

**Reply by Mattmjs** [reply]
Check your anti virus and firewall aren't blocking the companion app.

**Reply by spikerdc** [reply]
no interference from anti virus or firewall.  I decided to roll back Firefox on my Desktop and i can now download the videos again that i could not download before. I think its important to note 2 things .... first i could still download older videos with the most recent Firefox, and secondly on my laptop where i still have the current up to date Firefox i can download old or new videos.  So this seems to indicate some problem with Windows and/or Firefox and/or Downloader.  I'd still be interested in any input from anyone because i'm guessing that if i update Firefox again on my Desktop then i will no longer be able to download that site's current videos again. Thanks for any help everyone.

**Reply by Mattmjs** [reply]
If it's a site requiring a login, it's possible to still investigate it. Follow these steps carefully.
1. Click the 3 dots button next to download and look down to details.
2. Click on it and a page opens with more information.
3. Ctrl + A to select all on the page.
4. Ctrl + C to copy information to the clipboard.
5. Ctrl + V to paste the information here in a reply.

**Comment by spikerdc** [comment]
i contacted the person who owns the site and uploads the videos and he said
that he has changed nothing on his end

this is my latest reply below, basically saying that i reverted to the
previous Firefox on my Desktop and i can download any of the videos again,
but on my Laptop i do have the most current release of Firefox and i can
download any and all videos there. I don't know what this means as far as a
problem goes or how to fix it. My reply below:

no interference from anti virus or firewall. I decided to roll back Firefox
on my Desktop and i can now download the videos again that i could not
download before. I think its important to note 2 things .... first i could
still download older videos with the most recent Firefox, and secondly on
my laptop where i still have the current up to date Firefox i can download
old or new videos. So this seems to indicate some problem with Windows
and/or Firefox and/or Downloader. I'd still be interested in any input from
anyone because i'm guessing that if i update Firefox again on my Desktop
then i will no longer be able to download that site's current videos again.
Thanks for any help everyone.

**Comment by spikerdc** [comment]
Thank you for your response. I had reverted to an older version of Firefox
and it was working ok. But i updated again to the current version and
retested the site and i was now able to download videos again without
problem.

I still did what you recommended, and will attach it here below, but its a
moot point since upon updating Firefox again all seems well.
Thank you all for your help.
Respectfully.

Tab 3 (incognito: true)
Title Viper Trading
Page URL
https://member.vipertradingsystems.com/courses/products/90b48621-dca9-42c4-af4b-daf7eb1de1f5/categories/23ab70bd-dc55-413a-9295-b544507f4ccd/posts/83cb5113-3d42-4d15-8a38-c5e09872a463
with 12 headers
Thumbnail
https://cdn.courses.apisystem.tech/memberships/aBV5cTHTk7hmjrfQAPem/post/VTS-LOGO-VIPER-GREEN-3x-20-0c8i.jpg
Low Quality false
#0 main url
https://content.apisystem.tech/hls/v2/memberships/aBV5cTHTk7hmjrfQAPem/videos/46a36470-5295-4e97-9723-c91d4992df44_,53,32,16,00k.mp4.urlset/master.m3u8?token=eyJhbGciOiJSUzI1NiIsImtpZCI6ImI4Y2FjOTViNGE1YWNkZTBiOTY1NzJkZWU4YzhjOTVlZWU0OGNjY2QiLCJ0eXAiOiJKV1QifQ.eyJ1c2VyX2lkIjoid01TNmNWZlBnVmppWWpQbkt0ajUiLCJjb250YWN0X2lkIjoid01TNmNWZlBnVmppWWpQbkt0ajUiLCJyb2xlIjoidXNlciIsImxvY2F0aW9uSWQiOiJhQlY1Y1RIVGs3aG1qcmZRQVBlbSIsImV4dGVybmFsVXNlcklkIjoiYjhlODlkN2QtNjNmOS00N2FhLTgyYTYtMDQxYjUwNWYwZWU0IiwiYXBpS2V5IjoiZWUzNmU3NzYtZDBmZi00NjRiLWE5NjUtYWE3MmQ2YmRlMzAyIiwiaXNzIjoiaHR0cHM6Ly9zZWN1cmV0b2tlbi5nb29nbGUuY29tL2hpZ2hsZXZlbC1iYWNrZW5kIiwiYXVkIjoiaGlnaGxldmVsLWJhY2tlbmQiLCJhdXRoX3RpbWUiOjE3MzExNDg0MzAsInN1YiI6IndNUzZjVmZQZ1ZqaVlqUG5LdGo1IiwiaWF0IjoxNzMxMTQ4NDMwLCJleHAiOjE3MzExNTIwMzAsImZpcmViYXNlIjp7ImlkZW50aXRpZXMiOnt9LCJzaWduX2luX3Byb3ZpZGVyIjoiY3VzdG9tIn19.qXz-rh7-MpLNJPyhvFMrSP1TXeL5uwa---CODhd2eyJ8YKKtqqvb52BUmXiP5PKq6UF-siSQ5s1EgFD3Oz8ZzumXkOd-17uvo4VCZnXEjdeJZSc5AUxrmI5BqMA5ujNMwt85pdNsaIQ_gDAWX-NmFu0TUbdeW36scW7501FpDsDaUbFG_rld7pU1hUkRwnrkI4LhcUFK9AyyKDtlMtbs7Zn5O9HjfUr2S6I5o7VV5pC1UOoXUU9FHJ7Y1gH8_usgu0vWO4xUiO_XENYI16FSNc5dz19mxCYABtK1Ls2Hx32hJK7nNcPVKuhH1iNXwCCiHi1vDF-pUGIbvXpzvG_u5Q
#0 video component
https://content.apisystem.tech/hls/v2/memberships/aBV5cTHTk7hmjrfQAPem/videos/46a36470-5295-4e97-9723-c91d4992df44_,53,32,16,00k.mp4.urlset/index-f1-v1-a1.m3u8
#0 audio component none
#0 core media Mp4 b:Hls p:hls 3391.241s
#0 audio AAC at unknown
#0 video H264 at 339723, fps: 29.85, q:1080, {"width":1920,"height":1080}
#1 main url
https://content.apisystem.tech/hls/v2/memberships/aBV5cTHTk7hmjrfQAPem/videos/46a36470-5295-4e97-9723-c91d4992df44_,53,32,16,00k.mp4.urlset/master.m3u8?token=eyJhbGciOiJSUzI1NiIsImtpZCI6ImI4Y2FjOTViNGE1YWNkZTBiOTY1NzJkZWU4YzhjOTVlZWU0OGNjY2QiLCJ0eXAiOiJKV1QifQ.eyJ1c2VyX2lkIjoid01TNmNWZlBnVmppWWpQbkt0ajUiLCJjb250YWN0X2lkIjoid01TNmNWZlBnVmppWWpQbkt0ajUiLCJyb2xlIjoidXNlciIsImxvY2F0aW9uSWQiOiJhQlY1Y1RIVGs3aG1qcmZRQVBlbSIsImV4dGVybmFsVXNlcklkIjoiYjhlODlkN2QtNjNmOS00N2FhLTgyYTYtMDQxYjUwNWYwZWU0IiwiYXBpS2V5IjoiZWUzNmU3NzYtZDBmZi00NjRiLWE5NjUtYWE3MmQ2YmRlMzAyIiwiaXNzIjoiaHR0cHM6Ly9zZWN1cmV0b2tlbi5nb29nbGUuY29tL2hpZ2hsZXZlbC1iYWNrZW5kIiwiYXVkIjoiaGlnaGxldmVsLWJhY2tlbmQiLCJhdXRoX3RpbWUiOjE3MzExNDg0MzAsInN1YiI6IndNUzZjVmZQZ1ZqaVlqUG5LdGo1IiwiaWF0IjoxNzMxMTQ4NDMwLCJleHAiOjE3MzExNTIwMzAsImZpcmViYXNlIjp7ImlkZW50aXRpZXMiOnt9LCJzaWduX2luX3Byb3ZpZGVyIjoiY3VzdG9tIn19.qXz-rh7-MpLNJPyhvFMrSP1TXeL5uwa---CODhd2eyJ8YKKtqqvb52BUmXiP5PKq6UF-siSQ5s1EgFD3Oz8ZzumXkOd-17uvo4VCZnXEjdeJZSc5AUxrmI5BqMA5ujNMwt85pdNsaIQ_gDAWX-NmFu0TUbdeW36scW7501FpDsDaUbFG_rld7pU1hUkRwnrkI4LhcUFK9AyyKDtlMtbs7Zn5O9HjfUr2S6I5o7VV5pC1UOoXUU9FHJ7Y1gH8_usgu0vWO4xUiO_XENYI16FSNc5dz19mxCYABtK1Ls2Hx32hJK7nNcPVKuhH1iNXwCCiHi1vDF-pUGIbvXpzvG_u5Q
#1 video component
https://content.apisystem.tech/hls/v2/memberships/aBV5cTHTk7hmjrfQAPem/videos/46a36470-5295-4e97-9723-c91d4992df44_,53,32,16,00k.mp4.urlset/index-f2-v1-a1.m3u8
#1 audio component none
#1 core media Mp4 b:Hls p:hls 3391.241s
#1 audio AAC at unknown
#1 video H264 at 237614, fps: 29.85, q:720, {"width":1280,"height":720}
#2 main url
https://content.apisystem.tech/hls/v2/memberships/aBV5cTHTk7hmjrfQAPem/videos/46a36470-5295-4e97-9723-c91d4992df44_,53,32,16,00k.mp4.urlset/master.m3u8?token=eyJhbGciOiJSUzI1NiIsImtpZCI6ImI4Y2FjOTViNGE1YWNkZTBiOTY1NzJkZWU4YzhjOTVlZWU0OGNjY2QiLCJ0eXAiOiJKV1QifQ.eyJ1c2VyX2lkIjoid01TNmNWZlBnVmppWWpQbkt0ajUiLCJjb250YWN0X2lkIjoid01TNmNWZlBnVmppWWpQbkt0ajUiLCJyb2xlIjoidXNlciIsImxvY2F0aW9uSWQiOiJhQlY1Y1RIVGs3aG1qcmZRQVBlbSIsImV4dGVybmFsVXNlcklkIjoiYjhlODlkN2QtNjNmOS00N2FhLTgyYTYtMDQxYjUwNWYwZWU0IiwiYXBpS2V5IjoiZWUzNmU3NzYtZDBmZi00NjRiLWE5NjUtYWE3MmQ2YmRlMzAyIiwiaXNzIjoiaHR0cHM6Ly9zZWN1cmV0b2tlbi5nb29nbGUuY29tL2hpZ2hsZXZlbC1iYWNrZW5kIiwiYXVkIjoiaGlnaGxldmVsLWJhY2tlbmQiLCJhdXRoX3RpbWUiOjE3MzExNDg0MzAsInN1YiI6IndNUzZjVmZQZ1ZqaVlqUG5LdGo1IiwiaWF0IjoxNzMxMTQ4NDMwLCJleHAiOjE3MzExNTIwMzAsImZpcmViYXNlIjp7ImlkZW50aXRpZXMiOnt9LCJzaWduX2luX3Byb3ZpZGVyIjoiY3VzdG9tIn19.qXz-rh7-MpLNJPyhvFMrSP1TXeL5uwa---CODhd2eyJ8YKKtqqvb52BUmXiP5PKq6UF-siSQ5s1EgFD3Oz8ZzumXkOd-17uvo4VCZnXEjdeJZSc5AUxrmI5BqMA5ujNMwt85pdNsaIQ_gDAWX-NmFu0TUbdeW36scW7501FpDsDaUbFG_rld7pU1hUkRwnrkI4LhcUFK9AyyKDtlMtbs7Zn5O9HjfUr2S6I5o7VV5pC1UOoXUU9FHJ7Y1gH8_usgu0vWO4xUiO_XENYI16FSNc5dz19mxCYABtK1Ls2Hx32hJK7nNcPVKuhH1iNXwCCiHi1vDF-pUGIbvXpzvG_u5Q
#2 video component
https://content.apisystem.tech/hls/v2/memberships/aBV5cTHTk7hmjrfQAPem/videos/46a36470-5295-4e97-9723-c91d4992df44_,53,32,16,00k.mp4.urlset/index-f3-v1-a1.m3u8
#2 audio component none
#2 core media Mp4 b:Hls p:hls 3391.241s
#2 audio AAC at unknown
#2 video H264 at 170489, fps: 29.85, q:480, {"width":854,"height":480}

**Reply by Mattmjs** [reply]
The main url doesn't work but the video component url works, downloading a 56 minute video with the audio. I thought it might just give the video on it's own without the audio but that was not the case. All in it seems to work and for you too, must be an intermittent issue.

**Comment by spikerdc** [comment]
thank you again for your help
